lib/doorkeeper/rake/setup.rake

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+namespace :doorkeeper do
+  task setup: :environment do
```

Comment:
This is inspired by `resque`, where you have a hook that is executed before each rake task of that gem.

I decided to depend on `:environment` by default, but to be honest I never used Doorkeeper outside a Rails-context. If it is neccessary, we could also make the dependency on `environment` optional, as they've done in `resque`.

Reply:
Looks pretty good for me :+1: